packages/MarkdownEditor-Core.package/MarkdownCodeBlock.class/instance/asText.st

```diff
@@ -2,7 +2,8 @@ converting
 asText
 
 	| code result |
-	code := self smalltalkStyler styledTextFor: self codeContent asText.
+	code := self codeContent asText.
+	self smalltalkStyler privateStyle: code.
```

Comment:
I think it's weird to use a private method so maybe we should ask someone if this is how it's supposed to be used.

Reply:
It's approved officially, but should get a comment